frontend/tests/components/addMemberModal.ts

```diff
@@ -3,6 +3,7 @@ import { BaseComponent } from './baseComponent';
 
 const EMAIL_LABEL = 'Email';
 const ROLE_LABEL = 'Role';
+const INVITE_LABEL = 'Invite';
 const SUBMIT_BUTTON_LABEL = new RegExp(`(Add Member)|(Add or invite Member)`);
```

Comment:
It used to be Add or invite member, but I changed it to just Add Member since we have the invite checkbox now. Was this a right decision? @rmunn

Reply:
Yes, this is fine. By the way, the parentheses aren't necessary as `|` has the lowest precedence of any regexp. So this regex should be `Add Member|Add or invite Member`. Try it out at https://regex101.com/ (be sure to click the "ECMAScript / JavaScript" regex flavor on the left) and you'll see that the parentheses can be removed.

Also, if you have parentheses, then the regex captures the group inside parentheses for later use, which is unnecessary and wastes a small amount of memory if the group isn't needed. So removing them is a win for both optimization *and* code simplicity. That doesn't happen often; usually optimized code is harder to read, which is why the standard advice on optimization is as follows:

1. Don't do it.
2. Don't do it YET.

But when a change makes the code faster to run *and* easier to read, then you should just do it without thinking about it.